acados/ocp_nlp/ocp_nlp_sqp.h

```diff
@@ -64,11 +64,12 @@ typedef struct
     double tol_stat;     // exit tolerance on stationarity condition
     double tol_eq;       // exit tolerance on equality constraints
     double tol_ineq;     // exit tolerance on inequality constraints
-    double tol_comp;     // exit tolerance on complemetarity condition
+    double tol_comp;     // exit tolerance on complementarity condition
     int max_iter;
     int ext_qp_res;      // compute external QP residuals (i.e. at SQP level) at each SQP iteration (for debugging)
     int qp_warm_start;   // qp_warm_start in all but the first sqp iterations
     bool warm_start_first_qp; // to set qp_warm_start in first iteration
+    int phase;           // only phase 0 at the moment 
```

Comment:
Yes, for example, I need to revive zero-order NMPC with lifted integrators pretty soon (next PR) and it will use something similar to this field (or even the same). Parallelization of mulit-level iterations is yet another topic, but I haven't thought about it because I don't need it at the moment. Do you have some ideas about it?

Reply:
> I need to revive zero-order NMPC with lifted integrators pretty soon (next PR) and it will use something similar to this field (or even the same).

Sounds good!
I think as soon as you introduce another meaning for `phase` it would be really good to use `enum`

> Parallelization of mulit-level iterations is yet another topic, but I haven't thought about it because I don't need it at the moment. Do you have some ideas about it?

No, it remains interesting though :grinning:

Reply:
+1 for enum!